backend-api/src/functions/asyncCredential/asyncCredentialHandler.test.ts

```diff
@@ -236,7 +236,7 @@ describe("Async Credential", () => {
         });
 
         dependencies.tokenService = () =>
-          new MockTokenServiceGetDecodedTokenFailure();
+          new MockTokenServiceGetDecodedTokenErrorResult();
```

Comment:
We're using "*Success" in our naming of passing mocks. Is there are a reason why we're using a different naming convention for the failing mocks?

Reply:
I agree with using SuccessResult. I'll update those ones

Reply:
Updated